src/java.desktop/share/classes/java/awt/package-info.java

```diff
@@ -58,6 +58,7 @@
  *                  The Java AWT Native Interface (JAWT)</a>
  * </ul>
  *
+ * @spec AWT_Native_Interface.html The Java AWT Native Interface Specification and Guide
  * @since 1.0
```

Comment:
I wonder if links to html we include in the javadoc should be really treated in the same manner as referecnes to externally defined specifactions ?
But I also wonder why only the native_interface spec was added and not the other two ?